ecommerce/pricing/lib/pricing/promotions_calendar.rb

```diff
@@ -45,9 +45,8 @@ def initialize(start_time, end_time, discount)
       end
 
       def running?(timestamp)
-        (@start_time...@end_time).include?(timestamp)
+        (@start_time...@end_time).cover?(timestamp)
```

Comment:
The fix looks perfect.

Apparently, the previous tests didn't catch this problem. Let's add tests as part of this change/fix.

Reply:
Added, thank you! 👍